### ISSUE TITLE
Agents - Preview: Unexpected error. Please, report this error.

### ISSUE THREAD

**Issue by itcrowdsource** [wazuh/wazuh#360]
Ive just upgraded Wazuh to the latest version. Now I am getting three error messages in Kibana:

Agents - Preview: Unexpected error. Please, report this error.
Saved "field" parameter is now invalid. Please select a new field.
Visualize: "field" is a required parameter

Im unable to see my agents (only the # of agents) nor can I see the Wazuh dasboard properly because it is missing some parts.

I tried to reinstall the entire Wazuh stack but that did not help either. Also installed the latest version of the Wazuh-app itself via https://github.com/wazuh/wazuh-kibana-app

But all with no result.

**Comment by JuanjiJG** [comment]
Hi @itcrowdsource, and sorry for the late response.

The error `Saved "field" parameter is now invalid. Please select a new field.` after an upgrade happens if you did not insert the Wazuh templates and sample alert before starting **Logstash**. We recommend to carefully follow the steps from our [upgrading guide](https://documentation.wazuh.com/current/installation-guide/upgrading/index.html).

Please, execute the following commands and paste here the output:
`curl -XGET localhost:9200/_cat/indices?v`
`curl -XGET localhost:9200/_cat/templates?v`

These commands will **list your current templates and indices** from your Elasticsearch installation.

Best regards,
Juanjo

**Comment by smtszk** [comment]
"Unexpected error. Please, report this error"

When I saw this I checked API logs which is not efficent. In API configuration file, change the log level to "debug" then check API logs again.

My error is all about "local_rules.xml" file. 

There are some "<!--/--!>" comment errors. After deleting them, errors are gone.

Hope it helps.

**Comment by JuanjiJG** [comment]
Hi again @smtszk,

I'm glad you found the solution to your problem. Thanks for your feedback, we always read the community suggestions and we consider it for the future.

I'll report the team about that **"local_rules.xml"** error you experienced.

Related to the app, the next releases will have a better error reporting system, so it will be much easier to find what's the problem when something on the app fails.

Feel free to post again or create a new issue if you still have questions or problems about Wazuh.

Best regards,
Juanjo
